src/OrleansRuntime/MultiClusterNetwork/MultiClusterGossipChannelFactory.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+using Orleans.Runtime.Configuration;
+
+namespace Orleans.Runtime.MultiClusterNetwork
+{
+    internal class MultiClusterGossipChannelFactory
+    {
+        private readonly GlobalConfiguration globalConfig;
+        private readonly Logger logger;
+
+        public MultiClusterGossipChannelFactory(GlobalConfiguration globalConfig)
+        {
+            this.globalConfig = globalConfig;
+            logger = LogManager.GetLogger("MultiClusterGossipChannelFactory", LoggerType.Runtime);
+        }
+
+        internal async Task<List<IGossipChannel>> CreateGossipChannels()
```

Comment:
This method hasn't meaningfully changed. The only difference is that it no longer takes the `Silo` parameter, since the configuration is injected instead.